### PR TITLE
upload test file to eb

### PR DESCRIPTION
**Pull request by rizalfr**

### CODE REVIEW

File: locust/web.py [hunk 1]
```diff
@@ -338,6 +340,20 @@ def convert_csv_to_json():
...
+def upload_file():
+    path = request.form.get('upload_file_path')
+    python_file = request.files['python_file']
+    python_file_extension = os.path.splitext(python_file.filename)[1]
+    if not python_file and python_file_extension != ".py":
+        return "Try again with python file with .py extension"
+    try:
+        python_file.save(os.path.join((os.environ['PYTHONPATH'].split(os.pathsep))[-1] + path, python_file.filename))
+    except Exception:
+        logger.info("File is not uploaded. Note that all files deployed can not be overwritten. Try to change it to another name")
```
[the remark this answers is not in the document]
Comment (pancaprima) [reply locust/web.py]: also this message is bad

File: locust/static/style.css [hunk 2]
```diff
@@ -302,8 +302,13 @@ label[for="json_option"]+div{
     padding: 6px 15px 6px 0px !important;
 }
 
+#addNewFileModal .select2-selection--single {
```
Comment (pancaprima) [inline locust/static/style.css]: use hyphen delimited for css class name

File: locust/static/locust.js [hunk 3]
```diff
@@ -5,6 +5,15 @@ $(window).ready(function() {
     }
 });
 
+$("#locustfile").on('select2:close', function(){
+    event.preventDefault();
+    var el = $(this);
+    if(el.val()==="add_new_test_file") {
+        $('#addNewFileModal').modal('show');
```
[the remark this answers is not in the document]
Comment (rizalfr) [reply locust/static/locust.js]: done

File: locust/templates/index.html [hunk 4]
```diff
@@ -66,26 +66,25 @@
     </div>
     <div class="main">
         <div class="start" id="start">
-            <div style="position:relative;">
-                <a href="#" class="close_link">Close</a>
-            </div>
             <div class="padder">
                 <h2>Start new Locust swarm</h2>
-                <form action="/swarm" method="POST" id="swarm_form" class="start-timer">
+                <form action="/swarm" method="POST" id="swarm_form" class="start-timer upload_file">
```
[the remark this answers is not in the document]
Comment (rizalfr) [reply locust/templates/index.html]: it's not used anymore, deleted

**Pull request merged**